core/src/apps/wallet/get_address.py

```diff
@@ -1,11 +1,37 @@
+from trezor.crypto import bip32
 from trezor.messages import InputScriptType
 from trezor.messages.Address import Address
 
 from apps.common import coins
-from apps.common.layout import address_n_to_str, show_address, show_qr
+from apps.common.layout import address_n_to_str, show_address, show_qr, show_xpub
 from apps.common.paths import validate_path
 from apps.wallet.sign_tx import addresses
 
+if False:
+    from typing import List
+    from trezor.messages import HDNodeType
+    from trezor import wire
+    from apps.common.coininfo import CoinInfo
+
+
+async def show_xpubs(
+    ctx: wire.Context, coin: CoinInfo, pubnodes: List[HDNodeType]
+) -> bool:
+    for i, x in enumerate(pubnodes):
```

Comment:
I'm not very fond of these short variable names. `i` is fine, but I'd prefer calling `x` `pubnode` or similar.